app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::ItemsController < ApplicationController
+  def create
+    if item_params.keys.count == 4
+      item = Item.create(item_params)
+      render json: ItemSerializer.render(item), status: 201
+    else
+      render status: 422
+    end
+  end
+
+  private
+
+  def item_params
+    params.permit(:name, :description, :unit_price, :merchant_id)
```

Comment:
This is interesting. I've never seen the syntax of the individual keys not nested under a "parent" key. Is there a reason they're coming in like you have it here, or should it be:

```ruby
  params.require(:item).permit(:name, :description, :unit_price, :merchant_id)
```

Reply:
<img width="1080" alt="Screen Shot 2021-07-17 at 6 12 22 PM" src="https://user-images.githubusercontent.com/10294841/126052647-edabf6d7-9456-4a9f-be65-abff52040ba7.png">
maybe I'm interpreting this incorrectly?

Reply:
🤷‍♀️ If that’s what the spec says, then go with it!